app/models/publisher.rb

```diff
@@ -274,6 +275,17 @@ def publisher?
     role == PUBLISHER
   end
 
+  def default_site_banner
+    self.site_banners.find_by(id: self.default_site_banner_id)
+  end
+
+  def create_default_site_banner
+    headline = I18n.t 'banner.headline'
+    tagline = I18n.t 'banner.tagline'
+    default_site_banner = SiteBanner.create(publisher_id: self.id, channel_id: nil, title: headline, description: tagline, donation_amounts: [1, 5, 10], default_donation: 5, social_links: {youtube: '', twitter: '', twitch: ''})
```

Comment:
Nit, you can do this in one step:
```
self.default_site_banner = SiteBanner.create(publisher_id: self.id, channel_id: nil, title: headline, description: tagline, donation_amounts: [1, 5, 10], default_donation: 5, social_links: {youtube: '', twitter: '', twitch: ''})
```

Reply:
Moved this logic to a helper method, the reason for 2 lines instead of 1 is readability